src/main/java/org/isf/medicalstock/manager/MovStockInsertingManager.java

```diff
@@ -168,25 +168,25 @@ protected void validateMovement(Movement movement, boolean checkReference) throw
 
 			if (lot.getCode().length() >= 50) {
 				errors.add(new OHExceptionMessage(MessageBundle.getMessage("angal.common.error.title"),
-						MessageBundle.getMessage("angal.medicalstock.changethelotidbecauseitstoolong"), //$NON-NLS-1$
+						MessageBundle.getMessage("angal.medicalstock.thelotidistoolongmax50chars.msg"),
 						OHSeverityLevel.ERROR));
 			}
```

Comment:
should we use a format mask?

Reply:
I didn't do this one because it is only used once.